app/src/screens/Developer.tsx

```diff
@@ -237,6 +261,16 @@ const Settings: React.FC = () => {
 
   return (
     <SafeAreaView edges={['bottom', 'left', 'right']}>
+      <Modal
```

Comment:
Just to confirm my understanding, this modal will only appear in developer settings when they tap the logging option, yeah?

Reply:
Confirmed.